test/adoptoposs/jobs_test.exs

```diff
@@ -52,6 +52,36 @@ defmodule Adoptoposs.JobsTest do
       assert {:error, :unauthorized} = Bodyguard.permit(Jobs, :send_emails_weekly, date, 6)
       assert {:error, :unauthorized} = Bodyguard.permit(Jobs, :send_emails_weekly, date, 7)
     end
+
+    test "send_emails_biweekly is permitted for the first permitted weekday of the month", %{
+      date: date
+    } do
+      assert :ok = Bodyguard.permit(Jobs, :send_emails_biweekly, date, 2)
+      assert :ok = Bodyguard.permit(Jobs, :send_emails_biweekly, date, "2")
+    end
+
+    test "send_emails_biweekly is permitted on the permitted weekday for the second half of the month" do
+      {:ok, date} = Date.new(2222, 01, 15)
+
+      assert :ok = Bodyguard.permit(Jobs, :send_emails_biweekly, date, 2)
+      assert :ok = Bodyguard.permit(Jobs, :send_emails_biweekly, date, "2")
+    end
+
+    test "send_emails_biweekly is forbidden if the given date is not week 1 or week 3" do
+      # First authorized day within the first two weeks of the month
+      {:ok, date} = Date.new(2222, 01, 08)
+
+      assert {:error, :unauthorized} = Bodyguard.permit(Jobs, :send_emails_biweekly, date, 2)
+
+      assert {:error, :unauthorized} = Bodyguard.permit(Jobs, :send_emails_biweekly, date, "2")
+
+      # First authorized day within the first two weeks of the month
```

Comment:
Great tests! The only thing I’d have to remark is that I think the comments are a bit misleading.

Relating to the question on ranges for week 3 in my other comment –  could you add some test cases for the edge cases of the implementation to make sure we got the right day ranges?

That would be, it is forbidden for the: 
- permitted weekday, but if it is the first/last day of week 2
- permitted weekday, but if it is the first day of the week 4
- not permitted weekday in a permitted week

Reply:
@paulgoetze ,  added a few more test and reworded some of the comments.